vendors/tpl-systemes/vendor.yaml

```
@@ -0,0 +1,19 @@
+# ID of the vendor to be used as an internal reference for Actility vendors.
+# Define your own ID following this rule: Maximum of 8 characters in lowercase.
+# Be careful: Once you chose a vendor ID, it cannot be changed later
+id: tpl-systemes
```

```suggestion
id: tpl-sys
```
It should be maximum 8 characters. take a look at the comment above